Formula/chuck.rb

```diff
@@ -22,12 +22,6 @@ class Chuck < Formula
 
   depends_on xcode: :build
 
-  # Big Sur compile fix https://github.com/ccrma/chuck/pull/158
```

Comment:
Patch is definitely still needed.

Reply:
Agree, I did not see it in the tag. might need to check a bit

Reply:
Sorry, removed it as it didn't apply.

Reply:
Yep, it failed to apply because the line after the one the patch changes also changed. But the change in this patch is still needed.